src/i18n/content/jp/docs/integrations/infrastructure-integrations/get-started/introduction-infrastructure-integrations.mdx

```diff
@@ -219,7 +219,7 @@ Infrastructureインテグレーションは、[New Relicで使用](/docs/infras
         * イベントに添付された追加データ
         * メトリックスもしくはインベントリとは見なされないデータ
 
-        インテグレーションによっては、[その他のタイプの情報](https://docs.newrelic.co.jp/docs/integrations/new-relic-integrations/getting-started/understand-use-data-infrastructure-integrations#attributes)を属性として報告できます。
+        インテグレーションによっては、[その他のタイプの情報](/docs/telemetry-data-platform/understand-data/new-relic-data-types/)を属性として報告できます。
```

Comment:
Not sure we update japanese docs ourselves, check with Julien!

Reply:
I'll undo it!